src/components/metadata-modal/metadata-modal.js

```diff
@@ -74,19 +90,14 @@ const MetadataModal = ({ metadata, onToggle, visible }) => {
           <PreviewTable data={metadata.preview} size="large" />
         </div>
       )}
-      {!hasTable && (
-        <div className="pipeline-metadata-modal__bottom">
-          <button
-            className="pipeline-metadata-modal__collapse-plot"
-            onClick={onCollapsePlotClick}
-          >
-            <CollapseIcon className="pipeline-metadata-modal__collapse-plot-icon"></CollapseIcon>
-            <span className="pipeline-metadata-modal__collapse-plot-text">
-              {hasPlot
-                ? 'Collapse Plotly Visualization'
-                : 'Collapse Matplotlib Image'}
-            </span>
-          </button>
+      {hasJSON && (
+        <div className="pipeline-metadata-modal__preview-json">
+          <JSONObject
+            value={JSON.parse(metadata.preview)}
+            theme={theme}
+            style={{ background: 'transparent', fontSize: '15px' }}
+            collapsed={3}
```

Comment:
is the value 3 for collapsed a max depth value ? Is this always true to be 3 ?

Reply:
Yes, @jitu5 and I played around with it and found this was the best visually.